numba/np/math/__init__.py

```diff
@@ -0,0 +1,3 @@
+from numba.np.math import cmathimpl
+from numba.np.math import mathimpl
+from numba.np.math import numbers
```

Comment:
Seems unnecessary. Use-sites of these modules are already importing them explicitly.

Reply:
Fixed in [d17b0d8](https://github.com/numba/numba/pull/9437/commits/d17b0d8a3814d1b2b501366b83dcca2f1b75b3c6)